CHANGES_NEXT_RELEASE

```diff
@@ -10,3 +10,4 @@
   * sinon-chai
   * grunt and grunt related module
   * closure-linter-wrapper
+- Add Rule auto-naming on rule creation time
```

Comment:
I understand issue #307 should be mentioned.

Reply:
Yes, #307 should be mentioned. Is enough with these comments?

By the way... it could be that the description that I established in the CNR for this functionality is not the most correct. It would seem appropriate to you if I change it to something like `- Add 'ruleName' as variable automatically in rule text field on rule creation time`?

Reply:
Yes, I think is a good alternative. Maybe an explicit mention to EPL is clarifying. I mean in sum something like this:

```
- Add 'ruleName' as variable automatically in rule text field (EPL) on rule creation time (#208)
```

Reply:
Fixed in 0bfc9a0